src/Bicep.Core/Workspaces/IBicepSourceFile.cs

```diff
@@ -11,5 +12,13 @@ public interface IBicepSourceFile : ISourceFile
         ImmutableArray<int> LineStarts { get; }
 
         ProgramSyntax ProgramSyntax { get; }
+
```

Comment:
Feels like there are more similarities than differences between `BicepFile` & `BicepParamsFile` - what about an abstract base class rather than an interface for `IBicepSourceFile`?

Reply:
Yeah, it'll reuse the (extremely thin) implementation.